client/src/styles/global.css

```diff
@@ -0,0 +1,3 @@
+header {
+    background: #1b1b32;
+}
```

Comment:
Making this global creates a side-effect:
![image](https://user-images.githubusercontent.com/15801806/149797825-c7670118-b477-466a-9c03-cb66566ea864.png)

chakra-confirm uses `header` elements, so there's an interaction.

Is it possible to use css modules for everything, Header.tsx included?  At least until we've got a lot of repetition that might be better handling globally.

Reply:
Oops. Using css modules will work for header. Would you want to keep `global.css` empty for now, or remove it altogether?

Reply:
Good question.  It doesn't matter a great deal, but I think we should remove it.  Reason being, it's trivial to add if needed and, until then, it's tidier without it.